packages/website/pages/404.js

```diff
@@ -1,19 +1,14 @@
-import { useEffect } from 'react'
+// ===================================================================== Imports
+import GeneralPageData from '../content/pages/general.json';
+import BlockBuilder from '../components/blockbuilder/blockbuilder.js';
 
-import countly from '../lib/countly'
+// ===================================================================== Exports
+const blocks = GeneralPageData.error_404.sections;
 
-export default function Custom404() {
-    useEffect(() => {
-        countly.trackEvent(countly.events.NOT_FOUND, {
```

Comment:
@orvn do we track 404s in countly somewhere else? Looks like it's been removed from this 404 page.

Reply:
No, this wasn't intentional—thanks for catching. We had some issues with the 404 at one point and this particular Countly event needs to be added back (we did a sweep of Countly throughout the site to ensure everything was still in there, but this particular one must have been missed).